bin/alluxio-monitor.sh

```diff
@@ -0,0 +1,235 @@
+#!/usr/bin/env bash
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+LAUNCHER=
+# If debugging is enabled propagate that through to sub-shells
+if [[ "$-" == *x* ]]; then
+  LAUNCHER="bash -x"
+fi
+BIN=$(cd "$( dirname "$( readlink "$0" || echo "$0" )" )"; pwd)
+
+USAGE="Usage: alluxio-monitor.sh [-hL] ACTION [host1,host2,...]
```

Comment:
should we remove the `[host1,host2,...]`?

Reply:
Do you mean it's not necessary to expose this to the users?

Reply:
I thought the hosts were determined by the ACTION, nothing in the USAGE says anything about [host1,host2,...]

Reply:
Yes, I didn't added to the usage, my mistake, I will add the missing documentation.

Reply:
Done